### ISSUE TITLE
Accessibility: Link without a text alternative

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#525]
**Describe the bug**
[Link without a text alternative](https://my2.siteimprove.com/Accessibility/1081272/NextGen/Issue?ruleId=11&pageSegments=&issueKind=1&tagFilterType=2&conformance=&siteGoal=)

Links can be appended to blank spaces within text that has been added to pages. Content Owners need to edit and remove these empty links/spaces.

Exhibit: Chinese Traditional Dress
Page: https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=68309980197&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r11/failed/Z0R5T01zWG0=;N2Y3YWMwM2M=
Content: 45 instances of blank links

Exhibit: Fashion in Transit
Page: https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=68296818249&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r11/failed/UXJRcERsMUk=;MzIwZTE4YTc=
Content: 4 instances of blank links

Exhibit: CHOCOLATE: Food of the Gods
Page: https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=58323108344&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r11/failed/ZVpWN1dhaVE=;NTgwN2E5MjY=
Content: 2 instances of blank links

Exhibit: Do It Yourself, Do It Together
Page: https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=58331541395&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r11/failed/UGVFYzY4RGk=;ODc5MjRjNjM=
Content: 2 instances of blank links

Exhibit: Uris Library Cocktail Lounge
Page: https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=68298123874&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r11/failed/bmVNbEZseHc=;MWQyYThjZTU=
Content: 2 instances of blank links

Exhibit: A Fashionista's Guide to the Galaxy
Page: https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=58325421988&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r11/failed/SEdDaEU4VGU=;OTMzOTk2NjY=
Content: 1 instance of blank link
